app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -1,5 +1,14 @@
 <% content_for :title, @application_form.amendable? ? t('page_titles.edit_application_form') : t('page_titles.application_form') %>
 
+<% if FeatureFlag.active?('covid_19') && flash[:success].nil? %>
```

Comment:
It's possible to have different types of `flash`: `flash[:success]`, `flash[:warning]` and `flash[:info]` - see `app/components/flash_message_component.html.erb`. So I think this should be:

```suggestion
<% if FeatureFlag.active?('covid_19') && flash.empty? %>
```

Does that make sense?

Reply:
Yep that's right!